pytoda/smiles/polymer_language.py

```diff
@@ -94,15 +109,10 @@ def smiles_to_token_indexes(self, smiles: str) -> Indexes:
         Returns:
             Indexes: indexes representation for the SMILES/SELFIES provided.
         """
-        return [
-            self.token_to_index['<' + self.current_entity.upper() + '_START>']
-        ] + [
-            self.token_to_index[token]
-            for token in self.smiles_tokenizer(smiles)
-            if token in self.token_to_index
-        ] + [
-            self.token_to_index['<' + self.current_entity.upper() + '_STOP>']
-        ]
+        tokenized_smiles = self.add_start_stop_tokens(
+            self.smiles_tokenizer(smiles)
```

Comment:
Here, you change the default behavior of the function from skipping over unknown tokens (`if token in self.token_to_index`) to raising errors. Did you do this intentionally? I agree, the code can be improved, but I'm not sure we want this default behavior.

Reply:
Oh, shit, I think I overlooked this

Reply:
I'll revert to the previous behavior, but at the same time, I don't think that silently skipping tokens is the best solution. I'll put a note to revise that in the future (or open an issue).

Reply:
Revert pushed in 28303abe6e17a01d2a8c95075982aeaf5c2216ad